DocumentApp/api/Data/Repositories/DocsRepository.cs

```diff
@@ -21,8 +21,7 @@ public DocsRepository(DataContext context)
         public async Task<IEnumerable<DocDb>> GetDocsAsync()
         {
```

Comment:
`
 public Task<DocDb[]> GetDocsAsync()
 => _context.Docs.AsNoTracking()
            .Include(d => d.Category)        
            .ToArrayAsync();
`

Reply:
Got it